src/main/java/org/jenkinsci/test/acceptance/junit/JenkinsAcceptanceTestRule.java

```diff
@@ -56,6 +70,14 @@ public void evaluate() throws Throwable {
 
                 injector.injectMembers(this);
 
+                if (enableInstallWizard) {
```

Comment:
Yes the test is only going to be executed in the case of LocalController for all the other cases where we cannot control the start of the instance or ensure insolation the test will be skiped. See line 77